packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -335,8 +335,8 @@ def test_dcos_diagnostics_bundle_create_download_delete(dcos_api_session):
 def _check_diagnostics_bundle_status(dcos_api_session):
     # validate diagnostics job status response
     diagnostics_bundle_status = check_json(dcos_api_session.health.get('/report/diagnostics/status/all'))
-    required_status_fields = ['is_running', 'status', 'errors', 'last_bundle_dir', 'job_started', 'job_ended',
```

Comment:
Care to elaborate why this is necessary in the PR description?

Reply:
Added commit message

> In https://github.com/dcos/dcos-diagnostics/commit/43298bfb1c5af73a07f233bcd8cac757d7bd7514 we removed job_ended, duration, errors fields from bundle status when they are empty. This commit removes checking if bundle status contains that fields.